cpp/src/IceGrid/Scanner.l

```diff
@@ -213,7 +213,7 @@ namespace IceGrid
         keywordMap["describe"] = ICEGRID_DESCRIBE;
         keywordMap["properties"] = ICEGRID_PROPERTIES;
         keywordMap["property"] = ICEGRID_PROPERTY;
-        keywordMap["state"] = ICEGRID_STATE;
+        keywordMap["status"] = ICEGRID_STATUS;
```

Comment:
Should we still support the old `state` keyword for backwards compatibility?
If we wanted to, it would be as simple as adding this one line:
```suggestion
        keywordMap["status"] = ICEGRID_STATUS;
        keywordMap["state"] = ICEGRID_STATUS; // Kept for backwards compatibility
```
No need to muck around with the grammar or parser.